webapp/service_worker_handler.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package webapp
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+	"google.golang.org/appengine"
+)
+
+var (
+	swTemplate = template.Must(template.ParseFiles("templates/service-worker.js"))
+	// NOTE(lukebjerring): If tweaking service worker locally, change to
+	// sevenCharSHA, _ = regexp.Compile("^[0-9a-f]{7}|None$")
+	sevenCharSHA, _ = regexp.Compile("^[0-9a-f]{7}$")
+)
+
+func serviceWorkerHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := shared.NewAppEngineContext(r)
+	aeAPI := shared.NewAppEngineAPI(ctx)
+	if !aeAPI.IsFeatureEnabled("serviceWorker") {
```

Comment:
Ahh I see; it's behind a flag. I went ahead to create this flag in Datastore (on staging) and set it to true.